packages/react-percy-webpack/package.json

```diff
@@ -15,16 +15,16 @@
     "lib"
   ],
   "peerDependencies": {
-    "webpack": "^2.2.1"
+    "webpack": "^1.13 || 2.x"
```

Comment:
This basically just means that we support projects with webpack 1 or 2 (specifically 1.13 or newer, or any version of 2). Previously you'd get a warning if you tried to use React Percy in a project with webpack 1, even though it's entirely compatible.